src/server/routes.ts

```diff
@@ -1,12 +1,86 @@
-import express from "express";
+import express, { json } from "express";
 import cors from "cors";
+import Controller from "./controller/index";
+import { Trade } from "./types/index";
+
+const environment: string | undefined = process.env.PORT;
```

Comment:
**Comment**: This variable has no business being named `environment`.

Reply:
Renamed variable to envPort for clarification.